src/node_process_object.cc

```diff
@@ -196,28 +195,33 @@ void PatchProcessObject(const FunctionCallbackInfo<Value>& args) {
             .FromJust());
 
   // process.argv
-  process->Set(context,
-               FIXED_ONE_BYTE_STRING(isolate, "argv"),
-               ToV8Value(context, env->argv()).ToLocalChecked()).Check();
+  Local<Value> val;
+  if (!ToV8Value(context, env->argv()).ToLocal(&val) ||
+      !process->Set(context, FIXED_ONE_BYTE_STRING(isolate, "argv"), val)
+           .IsJust()) {
+    return;
+  }
 
-  // process.execArgv
-  process->Set(context,
-               FIXED_ONE_BYTE_STRING(isolate, "execArgv"),
-               ToV8Value(context, env->exec_argv())
-                   .ToLocalChecked()).Check();
+  if (!ToV8Value(context, env->exec_argv()).ToLocal(&val) ||
```

Comment:
Probably worth to keep the comment.

```suggestion
  // process.execArgv
  if (!ToV8Value(context, env->exec_argv()).ToLocal(&val) ||
```